config/locales/gems/pwned/en.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  errors:
+    messages:
+      not_pwned: you entered is insecure, please choose a stronger one
```

Comment:
```suggestion
      not_pwned: your password choice is insecure, please choose a stronger one
```

Reply:
GEM varsayılan olarak 'password' prefix'i koyduğu için bu şekilde. Yani ekrana basılırken şöyle görünüyor:

'Password you entered is insecure, please choose a stronger one'